src/components/popover/widgets/capture-keystroke/capture-keystroke-button.component.html

```diff
@@ -1,6 +1,7 @@
 <button type="button" class="btn btn-sm btn--capture-keystroke"
             [ngClass]="{'btn-default': !record, 'btn-info': record}"
-            (click)="start()"
+            (keydown.enter)="onButtonEnterDown($event)"
```

Comment:
`saveScanCode ` was called in both places so that you could save modifiers as well, without needing to press scan code. Your current implementation does not implement this, so you can't save `ctrl` for example or combination of `alt` and `ctrl`.

I think that this is not correct approach and we should be focused on fixing bugs (I didn't find any with current implementation) and implementing new things, not implementing new ones. If you don't understand my code please ask me, I am more then happy to explain it to you. Sometimes easier is not necessary better.

I think this PR should focus on implementing esc to close popover. I think that this can be done without radical change.